src/setup.ts

```diff
@@ -0,0 +1,91 @@
+import type {ListTablesCommandOutput} from '@aws-sdk/client-dynamodb/dist-types/commands/ListTablesCommand';
+import type {argValues} from 'dynamodb-local';
+import DynamoDbLocal from 'dynamodb-local';
+import {resolve} from 'path';
+import cwd from 'cwd';
+import {DynamoDB} from '@aws-sdk/client-dynamodb';
+import type {CreateTableCommandInput} from '@aws-sdk/client-dynamodb';
+import type {Config} from './types';
+import waitForLocalhost from './utils/wait-for-localhost';
+
+const debug = require('debug')('jest-dynamodb');
+
+const DEFAULT_PORT = 8000;
+const DEFAULT_HOST = 'localhost';
+const DEFAULT_OPTIONS: argValues[] = ['-sharedDb'];
+
+module.exports = async function () {
```

Comment:
Can we use ESM `import/export` instead of the `require/module.exports` here too?